README.md.gotmpl

```diff
@@ -1,39 +1,33 @@
-# Coder Helm Chart
+# Coder Helm Chart &middot; [![build](https://github.com/cdr/enterprise-helm/actions/workflows/build.yml/badge.svg)](https://github.com/cdr/enterprise-helm/actions/workflows/build.yml) [![Twitter Follow](https://img.shields.io/twitter/follow/CoderHQ?label=%40CoderHQ&style=social)](https://twitter.com/coderhq)
 
-⚠️ **WARNING**: This repository may not represent the current Coder release,
-since the default branch (master) may contain updates for an unreleased version
-of Coder.
+Coder moves developer workspaces to your cloud and centralizes their creation and management. Keep developers in flow with the power of the cloud and a superior developer experience.
 
-During Coder's release process, we inject version numbers and image references
-(e.g., `docker.io/coderenvs/coder-service:1.17.2`), then publish the result to
-our [Helm repository](https://helm.coder.com).
+The Coder chart is the best way to operate Coder on Kubernetes. It contains all the required components, and can scale to large deployments.
 
-When installing Coder, use the official packages that you can obtain as follows:
+![Coder Dashboard](./assets/coder.svg)
 
-```shell-session
-$ helm repo add coder https://helm.coder.com
-$ helm install coder coder/coder --namespace coder --version <VERSION_NUMBER>
-```
+## Getting Started
 
-See the [installation guide](https://coder.com/docs/setup/installation) for
-additional information.
+> ⚠️ **Warning**: This repository will not represent the latest Coder release. Reference
+our installation docs for instructions on a tagged release.
 
-If you have suggestions or feedback, please open a pull request or issue in this
-repository. If you have questions, please [contact support](#support).
+View [our docs](https://coder.com/docs/setup/installation) for detailed installation.
 
 ## Contributing
 
-The Helm templates for Coder services in the `templates` directory define the
-Kubernetes objects that comprise a Coder deployment. We list configurable values
-in the Helm values file (`values.yaml`), and we use `helm-docs` to generate the
-`README.md` file based on a template, `README.md.gotmpl`. When modifying content
-in the README or the `values.yaml` file, be sure to re-generate the README by
-running `make README.md`.
+Templates for Coder services live in the `templates` directory.
+Helm compiles templates with `values.yaml` when deploying.
+
+`README.md` is generated from `README.md.gotmpl` to ensure values are correct. Regenerate the readme:
+
+```bash
```

Comment:
For shell commands, use `shell-session` because it highlights slightly differently (the $ indicating the prompt)

I also am not sure why we put Contributing/Support sections above the values listing, I imagine that most people that come to this page will just want to see the available values they can use...

Reply:
Definitely agree. Going to move that section ordering.